src/java.base/share/classes/java/util/Comparator.java

```diff
@@ -155,8 +155,8 @@
      * {@link Object#equals(Object)}.  Additionally, this method can return
      * {@code true} <i>only</i> if the specified object is also a comparator
      * and it imposes the same ordering as this comparator.  Thus,
-     * {@code comp1.equals(comp2)} implies that {@code sgn(comp1.compare(o1,
-     * o2))==sgn(comp2.compare(o1, o2))} for every object reference
+     * {@code comp1.equals(comp2)} implies that {@code signum(comp1.compare(o1,
+     * o2))==signum(comp2.compare(o1, o2))} for every object reference
```

Comment:
Maybe make "signum" be a link here, similar to other locations where it's used.

Reply:
Okay -- I didn't want to go overboard making all the signum references links to the method, but I can change the first occurrence in Comparator.equals to a link.